src/ServiceControl.Audit.Persistence.SqlServer/SqlDbAuditDataStore.cs

```diff
@@ -10,7 +10,7 @@
     using ServiceControl.Audit.Monitoring;
     using ServiceControl.SagaAudit;
 
-    //TODO - not done as not sure on the table structure required here
+    //TODO! - not done as not sure on the table structure required here
```

Comment:
The whole thing is littered with todos, I propose to remove the persister completely for now